### ISSUE TITLE
[💡 Feature Request]: Multiple Voice Variants Support

### ISSUE THREAD

**Issue by TheMoodBoardz** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#643]
### What's the Problem?

Adding support for different voices. As you know google assistant has 10 main voices - including Red, Orange, Amber, Green, Cyan, Blue, Purple, Pink, British Racing Green, and Sydney Harbor Blue.

Adding support for these would bring the app experience higher as it would allow for more of a more custom google assistant.

### What's your solution or idea?

Adding this feature like it is on mobile [where it lists out as you click] (see screenshot) 

![Screenshot_20220201-144221](https://user-images.githubusercontent.com/59126098/152096085-075c55be-d938-49f4-9d69-83f5663241b6.jpg)
]

### Alternatives Considered

having a list that you could ask Google to run through [you click on it and it does not preview it unlike it does on mobile]

### App Version

v1.0.0-rc.2

### Additional context

This is a feature that I think many would appreciate (Keep up the good work) 

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%E2%9C%A8+enhancement%22+) and didn't find any duplicate issue
- [X] This issue only covers one particular feature request

**Comment by sumeshir26** [comment]
@TheMoodBoardz Did you try `Change my voice to [color]`

**Comment by TheMoodBoardz** [comment]
> @TheMoodBoardz Did you try typing out `Change your voice to [color]`

Yes. I thought it might work by doing that but it has no effect or will redirect you to youtube?

**Comment by sumeshir26** [comment]
Hmmm... Then this might be a limitation of the Developer Google Assistant SDK that this app uses.

**Comment by TheMoodBoardz** [comment]
> Change your voice to [color]

It can be done but it myself do not know how. Its a bit like how my watch has google assistant but is only able to use the red until on WiFi where it is Purple (if that makes sense)

**Comment by sumeshir26** [comment]
Try this:
- Go to https://console.actions.google.com/
- ![image](https://user-images.githubusercontent.com/68823982/155263493-52cebb37-4cb8-4cdf-a799-2d9cd0c51055.png)
- ![image](https://user-images.githubusercontent.com/68823982/155263525-b4e3dac6-24a9-4504-acd9-bd5c3492a7f6.png)
